### DISCUSSION TITLE
这是我见过的最屌的issue，没有之一，你们还见过更屌的可以发出来吃吃瓜

### DISCUSSION THREAD

**Discussion by WYC-2020** [coolsnowwolf/lede#11393] [Polls / 投票选举]
https://github.com/vernesong/OpenClash/issues/3393

**Comment by Zaphkito** [comment]
客观来说确实
主观来说GitHub官方本身就不允许某一个仓库存储了巨大的文件，就算是不限制总大小的二进制文件也要求一个文件不超过2g，而且这个问题确实有很多人提过了，我看他拖了这么久基本上没什么正面的回复就骂了
lede的源码以前也有个毛病我一直想说，但他现在已经改了
另外openclash有个很久之前的旧账，我不是很想提，反正看好自己的订阅文件

**Reply by iKira** [reply]
留后门上传别人的订阅文件？

**Comment by suisuyy** [comment]
确实太拽，这里有些人也是，别人问个问题 下面就一堆人指点，这也要提个issue赶紧关了，都不敢说话了 ，就不能好好讨论吗 ，这种可不是开源的风格 ，有问题你指出来 别人怎么改善就好，无用的责备只会劝退别人，不利于项目发展

**Comment by GreatMichaelLee** [comment]
哈哈，大佬都是孙悟空，小白都是小钻风。
